libmscore/layout.cpp

```diff
@@ -4315,20 +4353,23 @@ void LayoutContext::collectPage()
                         qreal margin = qMax(curSystem->minBottom(), curSystem->spacerDistance(false));
                         dist += qMax(margin, slb);
                         }
-                  breakPage = (y + dist) >= ey && breakPages;
+                  breakPage = (y + dist) >= ey && breakPages; // checks if the page is filled
                   }
             if (breakPage) {
                   qreal dist = qMax(prevSystem->minBottom(), prevSystem->spacerDistance(false));
                   dist = qMax(dist, slb);
                   layoutPage(page, ey - (y + dist));
                   // if we collected a system we cannot fit onto this page,
                   // we need to collect next page in order to correctly set system positions
-                  if (collected)
+                  if (collected) { // έχει γίνει κατάχρηση της collected
```

Comment:
Sorry, my Greek is not good enough to read this ;-)

Reply:
:laughing: the comments in Greek are my notes, they will be removed. This one says that I've abused `collected`.